docs/examples/evaluation/prometheus_evaluation.ipynb

```diff
@@ -0,0 +1,1498 @@
+{
```

Comment:
<p>4bit, or 8bit quantization?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9406/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
It is 4-bit quantization.

Reply:
There is 8-bit quantized model on different branch but HF is not allowing to create an endpoint with it.